docs/integrations/sources/openexchangerates.md

```diff
@@ -0,0 +1,48 @@
+# Open Exchange Rates
```

Comment:
I think we should rename this file to match the paths we've used in the connector itself

`open-exchange-rates.md`